README.md

```diff
@@ -42,6 +42,19 @@ SPSP_ENDPOINT=https://receiver-endpoint.com npm start
 #### SPSP_ENDPOINT
 * Type: String
 * Description: The receiver's [SPSP endpoint](https://interledger.org/rfcs/0009-simple-payment-setup-protocol/) to which SPSP queries are proxied.
+Mutually exclusive with `SPSP_ENDPOINTS_URL`.
+
+#### SPSP_ENDPOINTS_URL
+* Type: String
+* Description: URL used to fetch a receiver's [SPSP endpoint](https://interledger.org/rfcs/0009-simple-payment-setup-protocol/) to which an SPSP query is proxied.
+Mutually exclusive with `SPSP_ENDPOINT`.
+For each SPSP query, a GET request is sent to `SPSP_ENDPOINTS_URL` with the query's url path value (without the preceding slash) as the URI encoded `id` query parameter.
```

Comment:
An example would be great.

Reply:
:+1:  https://github.com/coilhq/receipt-verifier/pull/19/commits/c354a20eb27e4216175146a68b13838093e46850